app/screens/user_profile/user_profile.js

```diff
@@ -186,6 +190,46 @@ export default class UserProfile extends PureComponent {
         return null;
     };
 
+    buildCustomStatusBlock = () => {
+        const {formatMessage} = this.context.intl;
+        const {customStatus, theme, isMyUser} = this.props;
+        const style = createStyleSheet(theme);
+        const isStatusSet = customStatus.emoji || customStatus.text;
+
+        if (!isStatusSet) {
+            return null;
+        }
+
+        const label = formatMessage({id: 'user.settings.general.status', defaultMessage: 'Status'});
```

Comment:
how does this label help me differentiate between the presence status and the custom status?

Reply:
According to https://mattermost.atlassian.net/browse/MM-33149, the present status will be referred to as the user's availability and custom status will be referred to as status.